src/main/java/com/github/rodionovsasha/commentservice/exceptions/PermissionException.java

```diff
@@ -0,0 +1,11 @@
+package com.github.rodionovsasha.commentservice.exceptions;
+
+public final class PermissionException extends ServiceException {
```

Comment:
Rename to `TopicAccessException` maybe since `PermissionException` is too generic